packages/coinstac-common/src/services/classes/db-registry.js

```diff
@@ -188,14 +188,15 @@ class DBRegistry {
     conf.pouchConfig.ajax = result(this, 'ajax');
 
     /**
-     * Ensure local (client) instances' non-projects databases are stored in
-     * memory.
+     * Ensure local (client) instances' databases are stored in
+     * memory. This excludes projects and local consortium results.
      *
      * {@link https://github.com/MRN-Code/coinstac/issues/155}
      */
     if (
       this.localStores &&
       this.localStores.indexOf('projects') > -1 &&
+      this.localStores.indexOf('local-consortium') > -1 &&
```

Comment:
I foresee this causing issues as _local-consortium-xxx_ databases are “sync-out.” Local and remote databases could become out of sync, causing difficult state problems.